playbooks/adhoc/preflight/check.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: OSEv3
+  gather_facts: no
+  roles:
+    - openshift_preflight/facts
```

Comment:
@rhcarvalho I'm definitely open to adding a concept of sub-roles, I just wanted to avoid adding a new concept without having further discussion on it. I do worry a bit about it introducing some confusion, but I can definitely see it helping to clean up the directory structure and helping with role organization.

That said, I would expect that these roles would be able to be fully integrated into a 100% automated workflow without having to be called from distinct playbook. The mechanics of how that is accomplished is something that would need to be worked out, though.